### DISCUSSION TITLE
wolf-bites-tweets 2.0.0 开发小记 · TurboKre的博客 | wolf-bites-tweets 2.0.0 Development Notes · TurboKre's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ohmykreee/kreee-blog#9] [Comments]
# wolf-bites-tweets 2.0.0 开发小记

wolf-bites-tweets 2.0.0 开发小记 - A Kreee's Blog, powered by GitHub Pages, Hugo and Bilberry Premium Theme for Hugo.

https://blog.kre3.net/article/notes-of-developing-wbt-2.0.0/

**Comment by ohmykreee** [comment]
~~再测试一下新的 Giscus 评论系统~~

最近有点开坑开上头了😂
